docassemble/ALToolbox/data/static/review_widget.js

```diff
@@ -0,0 +1,56 @@
+var al_fade_speed = 300;
+
+function after_click() {
+  var thumbs_stage = document.getElementsByClassName('al-thumbs-widget');
+  var feedback_stage = document.getElementsByClassName('al-review-text');
+
+  for (let elem of thumbs_stage) {
+    $(elem).fadeOut(al_fade_speed);
+  }
+  setTimeout(function() {
+    for (let new_elem of feedback_stage) {
+      $(new_elem).fadeIn(al_fade_speed);
+    }
+  }, al_fade_speed);
+}
+
+function after_send() {
+  var feedback_stage = document.getElementsByClassName('al-review-text');
+  var all_done_stage = document.getElementsByClassName('al-after-review');
+
+  for (let elem of feedback_stage) {
```

Comment:
If there are indeed multiples of these elements, would a click on one send element hide all the other elements too?

Reply:
Nope, not designed for being used more than once on the same screen, I don't really see a use case there.